lib/src/tasks/coverage/exceptions.dart

```diff
@@ -38,10 +38,3 @@ class MissingLcovException implements Exception {
   final String message = missingLcovMessage;
   String toString() => 'MissingLcovException: $message';
 }
-
-class PortBoundException implements Exception {
-  final String message;
-  PortBoundException(String message, String port)
-      : this.message = 'Port $port could not be bound: $message';
-  String toString() => 'PortBoundException: $message';
-}
```

Comment:
I didn't really mean to remove this, but it didn't make sense in this exceptions file since it's not specific to coverage. It can certainly be re-added, would just need to live somewhere else.